client/cli/cli.go

```diff
@@ -63,27 +63,52 @@ func Run(run bool, pid int, args []string) {
 	}()
 
 	cmds := command.DebugCommands()
+	names := cmds.Names()
+
+	// set command completer
+	t.line.SetWordCompleter(func(line string, pos int) (head string, completions []string, tail string) {
+		start := strings.LastIndex(line[:pos], " ")
```

Comment:
I really like the idea of enhanced completion for breakpoint setting, especially auto completing function names, source files, etc..

Would you mind breaking the auto complete stuff out into a separate PR, because I'd like to merge that in by itself and have a place to discuss that feature.